lasy/profiles/from_openpmd_profile.py

```diff
@@ -110,13 +116,23 @@ def __init__(
         if not is_envelope:
             grid = create_grid(F, axes, dim, is_envelope=is_envelope)
             grid, omg0 = field_to_envelope(grid, dim, phase_unwrap_nd)
-            array = grid.get_temporal_field()[0]
+            array = grid.get_temporal_field()
         else:
-            s = io.Series(path + "/" + prefix + "_%T.h5", io.Access.read_only)
+            if is_waket:
+                filepath = path + "/" + prefix + "%T.h5"
```

Comment:
This maybe should be fixed in Wake-T.

Reply:
Here `is_waket` is only used to adapt the filepath. 
A general solution would be to let the user chose the file naming scheme, e.g., by passing directly `filepath` instead of `prefix`.